inline-r/src/Foreign/R.hsc

```diff
@@ -192,14 +187,25 @@ C.include "<stdint.h>"
 car :: SEXP s a -> IO (SomeSEXP s)
 car (unsexp -> s) = somesexp <$> [C.exp| SEXP { CAR( $(SEXP s) ) } |]
 
+-- | Set the CAR value and return it.
+setCar :: SEXP s a -> SEXP s b -> IO (SEXP s b)
+setCar (unsexp -> s) (unsexp -> s') = sexp <$> [C.exp| SEXP { SETCAR( $(SEXP s), $(SEXP s') ) } |]
```

Comment:
I'm guessing that you'd want to use `Language.C.Inline.Unsafe.exp` in these cases.

Reply:
Good point. Although that's true of a lot of functions in this module, none of which use `Language.C.Inline.Unsafe`. I suggest we address this in a followup.